ViteMaDose/Resources/Localization/fr.lproj/Localizable.strings

```diff
@@ -38,21 +42,40 @@
 "location.unavailable_date" = "Date indisponible";
 "location.unavailable_name" = "Nom du centre indisponible";
 "location.unavailable_address" = "Adresse indisponible";
+
 // Location start following
 "location.start_following_title" = "Recevoir les notifications ?";
 "location.start_following_message" = "Vous pouvez choisir d'être notifié pour chaque changement de disponibilité ou seulement lorsque des Chronodoses sont disponibles pour ce centre.";
 "location.follow_action_title" = "Suivre";
 "location.notify_button" = "Être notifié";
 "location.follow_button" = "Suivre sans notifications";
+
 // Location stop following
 "location.stop_following_title" = "Ne plus suivre ce centre ?";
 "location.stop_following_message" = "Si vous avez activé les notifications pour ce centre, elles seront désactivées.";
 "location.unfollow_action_title" = "Ne plus suivre";
 "location.stop_following_button" = "Ne plus suivre";
+
 // Error
 "error.generic.title" = "Oups! Nous avons des difficultés techniques...";
 "error.generic.retry_button" = "Réessayer";
 "error.generic.cancel_button" = "Annuler";
 "error.generic.default_message" = "Si le problème persiste, merci de contacter l'équipe sur https://covidtracker.fr/contact";
 "error.network.server_error" = "Nous rencontrons des problèmes avec le serveur, veuillez réessayer plus tard.";
 "error.network.offline" = "Il semblerait que vous soyez hors ligne.";
+
+// MARK: - Accessibility (Voice Over)
+
+"a11y.voiceover.home_screen.search_field" = "Tappez deux fois pour sélectionner ou rechercher un lieu en France.";
+"a11y.voiceover.home_screen.recent_searches" = "Recherches récentes.";
+"a11y.voiceover.home_screen.see_department_results" = "Voir les lieux dans le département ";
+"a11y.voiceover.home_screen.display_places_on_map" = "Tappez deux fois pour afficher la carte via le site web de Covid Tracker";
+"a11y.voiceover.navigation.back_button" = "Retour à l'écran principal";
+"a11y.voiceover.actions.call_button" = "Tappez deux fois pour appeler à ce numéro";
+"a11y.voiceover.actions.booking_button" = "Tappez deux fois pour prendre rendez-vous par site web";
+"a11y.voiceover.locations.see_places" = "Voir les lieux dans le département de %@";
```

Comment:
```suggestion
"a11y.voiceover.locations.see_places" = "Voir les lieux pour %@";
```
It's not always a department, happy to use something else if you have a better suggestion though :)